packages/notifi-frontend-client/lib/client/ensureSource.ts

```diff
@@ -1009,6 +1012,9 @@ export const ensureSourceAndFilters = async (
     case 'label': {
       throw new Error('Unsupported event type');
     }
+    case 'fusionMultiThreshhold': {
+      throw new Error('Unsupported event type');
+    }
```

Comment:
Here inside the ensureSourceAndFilters, we might also want to implement a `getFusionMultiThreshold` method to gather the correct filter and filterOptions.